compiler/rustc_target/src/spec/aarch64_unknown_uefi.rs

```diff
@@ -0,0 +1,36 @@
+// This defines the amd64 target for UEFI systems as described in the UEFI specification. See the
+// uefi-base module for generic UEFI options. On aarch64 systems UEFI systems always run in long-mode,
+// have the interrupt-controller pre-configured and force a single-CPU execution.
+//
+// The win64 ABI is used. It differs from the sysv64 ABI, so we must use a windows target with
+// LLVM. "aarch64-unknown-windows" is used to get the minimal subset of windows-specific features.
```

Comment:
These comments are copy-pasted from `x86_64_unknown_uefi.rs` and it's likely that they don't make sense for AArch64, "amd64" and "long-mode" are x86-specific terms in particular.